wireapp/wire-android-reloaded
# feat: display on premises BE info in welcome screen

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

display on-premises BE info in welcome screen


### Solutions

if the current auth server is on-premises server the server domain will be displayed in the welcome screen

https://user-images.githubusercontent.com/9083456/186651241-6e1354ba-e05b-4727-8bb7-3c31d27c4a04.mp4

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

open the app -> there should be no server info displayed in the welcome screen
click on a deep link -> app shows the custom server info

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
